chef/cookbooks/crowbar/templates/default/puma.rb.erb

```diff
@@ -0,0 +1,50 @@
+ROOT = File.expand_path("../../", __FILE__)
```

Comment:
This file is already totally fine, but maybe we should put it directly into `crowbar_framework/config`. We don't need to write it with chef.